data-exporter/src/main/java/org/reactome/release/dataexport/UCSC.java

```diff
@@ -209,14 +209,14 @@ private String getNoEventsErrorMessage(UniProtReactomeEntry uniProtReactomeEntry
 				"-[:referenceDatabase]->(rd:ReferenceDatabase)",
 				"WHERE ewas.speciesName IN ['Homo sapiens', 'Rattus norvegicus', 'Mus musculus'] AND rd.displayName =" +
```

Comment:
Why only these three species? Should this list be configurable?

Reply:
I'm not sure -- this was the way it was in the Perl script which predates my starting with Reactome.  I can e-mail Peter and Robin about it to ask.

Reply:
It might be that UCSC only handles these three, in which case a comment stating that should be enough. Or a constant string named "UCSC_ACCEPTED_SPECIES"... either should be fine.

Reply:
The named constant sounds good.  I'll do that and verify the accepted species with Peter and Robin as well.